app/controllers/people_controller.rb

```diff
@@ -2,15 +2,17 @@ class PeopleController < PublicFacingController
   enable_request_formats show: [:atom]
 
   def show
-    @person = PersonPresenter.new(Person.friendly.find(params[:id]), view_context)
-
     respond_to do |format|
       format.html do
+        @person = PersonPresenter.new(Person.friendly.find(params[:id]), view_context)
         set_meta_description("Biography of #{@person.name}.")
         set_slimmer_organisations_header(@person.organisations)
         set_slimmer_page_owner_header(@person.organisations.first)
       end
-      format.atom
+
+      format.atom do
+        redirect_to "/government/announcements?people[]=#{params[:id]}"
```

Comment:
does this need to be announcements.atom ?

Reply:
🤕